rmc/utils/constraint.py

```diff
@@ -509,7 +511,16 @@ def create_constraint_prep_ht(
     :param overwrite: Whether to overwrite Table. Default is True.
     :return: None; writes Table to path.
     """
-    ht = filtered_context.ht()
+    #TODO: Make a resource path for this table.
+    # ht = filtered_context.ht()
+    ht = hl.read_table(
+        "gs://gnomad/v4.1/constraint_an_coverage_corrected/apply_models/transcript_consequences/gnomad.v4.1.per_variant_expected.coverage_corrected.ht"
+        )
+    # Annotate obs and exp columns to be used downstream
+    ht = ht.annotate(
+        observed=ht.calibrate_mu.observed_variants[0],
```

Comment:
this is a naive question since I haven't thoroughly dug through Julia's code -- is this the correct field? this reads like these are the observed variants once filtered to high coverage sites only (and we want obs across all sites). if this table doesn't have the correct field, then we may want to add `ht = add_obs_annotation(ht)` here